tests/io/test_plugins.py

```diff
@@ -111,3 +111,15 @@ def test_plugins_list_success(api):
 def test_plugin_iterator_populate_family_cache(api):
     '''test for _populate_family_cache in PluginIterator'''
     getattr(PluginIterator(api), '_populate_family_cache')()
+
+
+@pytest.mark.vcr()
+def test_populate_family_cache_with_maptable(api):
```

Comment:
Just a minor suggestion, change test-case name to test_plugins_populate_family_cache_with_maptable

Reply:
Addressed comment